docs/source/users.rst

```diff
@@ -1,7 +1,7 @@
 User Management
 ===============
 
-This section covers the customer owner, adding and removing users, email on-boarding and user preferences.
+This section covers the customer owner, adding and removing users.
 
 Definitions
```

Comment:
Should the `User Preferences` section below be removed? (Lines 22 - 32)

Reply:
yep. done.